pkgs/development/libraries/md4c/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "md4c";
+  version = "release-0.4.6";
```

```suggestion
  version = "0.4.6";
```
tag prefixes don't belong in version field